src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -861,6 +862,7 @@ grpc_error_handle Chttp2ServerAddPort(Server* server, const char* addr,
   std::vector<grpc_error_handle> error_list;
   // Using lambda to avoid use of goto.
   grpc_error_handle error = [&]() {
+    grpc_error_handle error = GRPC_ERROR_NONE;
```

Comment:
same question? why is this needed? how was it working earlier?

Reply:
Ditto with above.